Create IDE literal expression documentation (#2045)
astronomer/docs: astro/cloud-ide/run-sql.md

7. In the **Select Connection** list, select the connection for the database you want to query against.

> 8. If you're using Warehouse SQL cells, configure the following additional fields to define your output table: 

Author: Sure, I assume that if a user doesn't input this then we create a temp table in their warehouse?

> 8. If you're using Warehouse SQL cells, configure the following additional fields to define your output table: 

Reviewer: Yep, that's right!

    - **Database**: The database for your output table
    - **Schema**:  The schema for your output table
    - **Table name**: The name of your output table

>     By default, your input is formatted as a literal string and is defined in your DAG with quotation marks. To use a Python expression to define any of these values, click **Python expression** next to the value name. If you already entered a literal string before changing this setting, remove the quotation marks that the IDE adds to your Python expression.

Author: ```suggestion
    By default, your input is formatted as a literal string and is defined in your DAG with quotation marks. To format your input as a Python expression, click **Python expression** next to the value name. If you already entered a literal string before changing this setting, remove the quotation marks that the IDE adds to your Python expression.
```

>     After you change this setting, your input for the value is defined in the DAG as a Python expression without quotation marks. You can use your input to call any Python value in your pipeline, such as a global import, an Airflow variable, or an environment variable. 

Reviewer 2: It wasn't clear to me if we turned on the Python expression setting or turned it off - so instead of referring to it just as "changed" i needed something a little more specific.
